PluginsAndFeatures/azure-toolkit-for-intellij/build.gradle

```diff
@@ -145,6 +145,9 @@ apply plugin: 'java'
 
 dependencies {
     compile 'com.nimbusds:oauth2-oidc-sdk:7.1.1', { force = true }
+    compile 'com.squareup.okhttp3:okhttp:4.9.1' , { force = true }
+    compile 'com.squareup.okhttp3:okhttp-urlconnection:4.9.1' , { force = true }
```

Comment:
we also need to exclude other versions of okhttp.

Reply:
done